go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/cosmos/cosmos-proto v1.0.0-beta.2
 	github.com/cosmos/cosmos-sdk v0.47.2
 	github.com/cosmos/go-bip39 v1.0.0
-	github.com/cosmos/gogoproto v1.4.10
+	github.com/cosmos/gogoproto v1.4.8
```

Comment:
This is done in order to avoid the following errors when running the `desmos query` command: 
```
WARNING: proto: file name query.proto does not start with expected testdata/; please make sure your folder structure matches the proto files fully-qualified names
WARNING: proto: file name testdata.proto does not start with expected testdata/; please make sure your folder structure matches the proto files fully-qualified names
WARNING: proto: file name tx.proto does not start with expected testdata/; please make sure your folder structure matches the proto files fully-qualified names
WARNING: proto: file name unknonwnproto.proto does not start with expected testdata/; please make sure your folder structure matches the proto files fully-qualified names
```

This is caused by how the Protobuf generation is setup inside Cosmos SDK. See [here](https://github.com/cosmos/cosmos-sdk/pull/16030#pullrequestreview-1413555965).